doc/cascadia/SettingsSchema.md

```diff
@@ -142,6 +142,7 @@ For commands with arguments:
 | `scrollUp` | Move the screen up. | | | |
 | `scrollUpPage` | Move the screen up a whole page. | | | |
 | `scrollDownPage` | Move the screen down a whole page. | | | |
+| `sendInput` | Sends some text input to the shell. | `input` | string | The text input to feed into the shell.<br>ANSI escape sequences may be used. Escape codes like `\x1b` must be written as `\u001b`.<br>For instance the input `"text\n"` will write "text" followed by a newline. `"\u001b[D"` will behave as if the left arrow button had been pressed. |
```

Comment:
Does this classify as "Documentation updated"? Do I need to submit a PR to the [docs repo](https://github.com/MicrosoftDocs/terminal)?

Reply:
Unfortunately, you do!